pvactools/lib/top_score_filter.py

```diff
@@ -132,4 +37,312 @@ def parser(cls, tool):
                  + "median: Use the median MT Score (i.e. the median MT ic50 binding score of all chosen prediction methods)."
         )
+        if tool == 'pvacseq':
+            parser.add_argument(
+                "--maximum-transcript-support-level", type=int,
+                help="When determining the top peptide, only consider those entries that meet this threshold for the Ensembl transcript support level (TSL). "
+                     + "Transcript support level needs to be <= this cutoff to be considered.",
+                default=1,
+                choices=[1,2,3,4,5]
+            )
+            parser.add_argument(
+                '-b', '--binding-threshold', type=int,
+                help="When determining the top peptide, only peptides passing the anchor criteria are considered. This criteria is failed if "
+                     + "all mutated amino acids of a peptide (Pos) are at an anchor position and the WT peptide has good binding (IC50 WT < binding_threshold).",
+                default=500
+            )
+            parser.add_argument(
+                '--allele-specific-binding-thresholds',
+                help="When determining the top peptide and evaluating the anchor criteria, use allele-specific binding thresholds. "
+                     + "If an allele does not have a special threshold value, the `--binding-threshold` value will be used.",
+                default=False,
+                action='store_true',
+            )
+
+            parser.add_argument(
+                "--allele-specific-anchors",
+                help="When determining the top peptide and evaluating the anchor criteria, use allele-specific anchor positions. This option "
+                     + "is available for 8, 9, 10, and 11mers and only for HLA-A, B, and C alleles. If this option is "
+                     + "not enabled or as a fallback for unsupported lengths and alleles, the default positions of 1, "
+                     + "2, epitope length - 1, and epitope length are used. Please see https://doi.org/10.1101/2020.12.08.416271 "
+                     + "for more details.",
+                default=False,
+                action='store_true',
+            )
+            parser.add_argument(
+                "--anchor-contribution-threshold", type=float_range(0.5,0.9),
+                help="For determining the top peptide and evaluating the anchor criteria using allele-specific anchors, each position is assigned a score based on how binding is "
+                     + "influenced by mutations. From these scores, the relative contribution of each position to the "
+                     + "overall binding is calculated. Starting with the highest relative contribution, positions whose "
+                     + "score together account for the selected contribution threshold are assigned as anchor locations. "
```

Comment:
```suggestion
                     + "scores together account for the selected contribution threshold are assigned as anchor locations. "
```

Reply:
I resolved this in a separate commit and fixed this grammar error in a couple additional places.